skimage/measure/_regionprops.py

```diff
@@ -358,6 +407,94 @@ def __eq__(self, other):
         return True
 
 
+def _props_to_dict(regions, properties=('label', 'bbox'), separator='-', always_include_label=True):
+    """Convert image regions properties into a dictionary
+
+    Parameters
+    ----------
+    regions : (N,) list
+        List of _RegionProperties objects as returned by :func:`regionprops`
```

Comment:
While we're at it, let's also fix the docstring of `regionprops` which currently refers to `RegionProperties` instead of `_RegionProperties`.